index.html

```diff
@@ -108,13 +108,51 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________*END TEMPLATE* Contributor card END ________  -->
 
-        
         <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ Julien's card START ________  -->
+        <div class="card">
+          <p class="name">Julien Mihai</p>
+          <p class="contact">
+            <i class="fab fa-facebook"></i>
+            <a href="https://www.facebook.com/mihaijulien" target="_blank">JulienMihai</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/mihaijulien" target="_blank">mihaijulien</a>
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/mihaijulien/" target="_blank">JulienMihai</a>
+          </p>
+          <p class="about">
+            Interested in tech, currently focused on machine learning.
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://pluralsight.com/courses/angular-fundamentals" target="_blank" title="Pluralsight">
+                  Angular Fundamentals
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://addyosmani.com/resources/essentialjsdesignpatterns/book/"
+                  target="_blank"
+                  title="JS Patterns"
+                >
+                  Learning JavaScript Design Patterns
+                </a>
+              </li>
+              <li>
+                <a href="https://developer.mozilla.org/en-US/docs/Learn" target="_blank" title="MDN Web Docs">
+                  MDN Learning
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
 
```

Comment:
You are missing the "Contributors card End" comment line. Add this to your code and make sure there is a blank line between your last comment line and the first comment line of the contributor below you.